File: rest/call-feedback/instance-get-example-1/instance-get-example-1.6.x.py
# Download the Python helper library from twilio.com/docs/python/install
import os
from twilio.rest import Client

# Your Account Sid and Auth Token from twilio.com/user/account
# To set up environmental variables, see http://twil.io/secure
account_sid = os.environ['TWILIO_ACCOUNT_SID']
auth_token = os.environ['TWILIO_AUTH_TOKEN']
client = Client(account_sid, auth_token)

feedback = client.calls("CAe03b7cd806070d1f32bdb7f1046a41c0") \
                 .feedback() \
                 .fetch()

print(feedback.date_created)

File: rest/accounts/instance-get-example-1/instance-get-example-1.6.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio.Rest.Api.V2010;
using Twilio;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Auth Token at twilio.com/console
        // To set up environmental variables, see http://twil.io/secure
        const string accountSid = Environment.GetEnvironmentVariable("TWILIO_ACCOUNT_SID");
        const string subAccountSid = Environment.GetEnvironmentVariable("TWILIO_SUBACCOUNT_SID");
        const string authToken = Environment.GetEnvironmentVariable("TWILIO_AUTH_TOKEN");
        TwilioClient.Init(accountSid, authToken);

        var account = AccountResource.Fetch(Environment.GetEnvironmentVariable(subAccountSid));

        Console.WriteLine(account.DateCreated);
    }
}
